### ISSUE TITLE
New (1.900.25+) Jasper build fail.

### ISSUE THREAD

**Issue by aerorahul** [NOAA-EMC/wgrib2#8]
Unable to build with Newer jasper.

```
/Users/rmahajan/scratch/NCEPLIBS/nceplibs/build/wgrib2/src/wgrib2/wgrib2/enc_jpeg2000_clone.c:133:10: error:
      no member named 'inmem_' in 'jas_image_t'
   image.inmem_=1;
   ~~~~~ ^
1 error generated.
make[6]: *** [wgrib2/CMakeFiles/obj_lib.dir/enc_jpeg2000_clone.c.o] Error 1
make[5]: *** [wgrib2/CMakeFiles/obj_lib.dir/all] Error 2
make[4]: *** [all] Error 2
make[3]: *** [wgrib2/src/wgrib2-stamp/wgrib2-build] Error 2
make[2]: *** [CMakeFiles/wgrib2.dir/all] Error 2
make[1]: *** [CMakeFiles/wgrib2.dir/rule] Error 2
make: *** [wgrib2] Error 2
```

**Comment by climbfuji** [comment]
Wrong Jasper if I remember correctly - use 1.900.1 or apply the usual patch to Jasper 2.x. Funny thing is, this offending functionality is never used in our applications.

**Comment by aerorahul** [comment]
@kgerheiser says to just comment out that line.
Doing that wgrib2 builds.
Infact -- looking at L1411 in `wgrib2/Changes`:
```
wgrib2/Changes:		enc_jpeg2000_clone.c:  jasper v1.900.25+ changes ABI, need to comment out line image.inmem_=1
```

So I think a bugfix is in order.

**Comment by aerorahul** [comment]
closed via #9
